consensus/reactor.go

```diff
@@ -1151,7 +1151,14 @@ func (ps *PeerState) SendCommit(commit *types.Commit) bool {
 func (ps *PeerState) PickSendVote(votes types.VoteSetReader) bool {
 	if vote, ok := ps.PickVoteToSend(votes); ok {
 		msg := &VoteMessage{vote}
-		ps.logger.Debug("Sending vote message", "ps", ps, "vote", vote)
+		ps.logger.Debug("Sending vote message",
```

Comment:
wdyt about using the formatting for multiline argument like this https://github.com/dashevo/tenderdash/blob/a3a43f4a7c8b9703618f0bc88476830a01366775/consensus/state.go#L875-L880
?
the idea is to put the first argument on a separate line without exception